sources/packages/backend/libs/services/src/disbursement-overaward/disbursement-overaward.service.ts

```diff
@@ -76,6 +78,32 @@ export class DisbursementOverawardService {
     return result;
   }
 
+  /**
+   * Adds a legacy overaward value to disbursement overawards table.
+   * @param studentId id of the student.
+   * @param overawardValue overaward value.
+   * @param disbursementValueCode disbursement value code.
+   * @param entityManager entity manager used to perform the query.
+   */
+  async addLegacyOveraward(
+    studentId: number,
+    overawardValue: number,
+    disbursementValueCode: string,
+    entityManager?: EntityManager,
+  ): Promise<void> {
+    const repo =
+      entityManager?.getRepository(DisbursementOveraward) ??
+      this.disbursementOverawardRepo;
+    const auditUser = await this.systemUsersService.systemUser();
+    await repo.insert({
+      student: { id: studentId } as Student,
+      disbursementValueCode,
+      overawardValue,
+      originType: DisbursementOverawardOriginType.LegacyOveraward,
+      creator: auditUser,
+    } as DisbursementOveraward);
```

Comment:
IMO, the cast (`as DisbursementOveraward`) is not bringing any benefit in this scenario.